isthmus/src/test/resources/tpch/queries/15.sql

```diff
@@ -1,4 +1,3 @@
--- converted to CTE since DDL is not part of Substrait.
```

Comment:
My preference would be to figure out why comments are causing a problem

Reply:
My guess is Isthmus will treat the dash here as input parameter and it throws `Missing required parameter: '<sql>'` message.

Reply:
Please file an issue for the comment in the sql file breaking when using native image